internal/pkg/api/middleware/auth_test.go

```
@@ -144,35 +144,35 @@ func TestCORSMiddleware(t *testing.T) {
 	t.Run("CORSHeaders", func(t *testing.T) {
 		allowedOrigin := "http://localhost:3000"
 		middleware := CORS(allowedOrigin)
-		
+
 		handler := middleware(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			w.WriteHeader(http.StatusOK)
 			w.Write([]byte("OK"))
```

golangci-lint (1.64.8)

Error return value of `w.Write` is not checked

(errcheck)